lncfg/address.go

```diff
@@ -121,6 +121,12 @@ func IsLoopback(addr string) bool {
 	return false
 }
 
+// isIPv6Host returns true if the host is IPV6 and false otherwise.
+func isIPv6Host(host string) bool {
```

Comment:
This should use [`ParseIP`](https://golang.org/pkg/net/#ParseIP) so it can properly handle identifying the `IPv6` format. This is less error prone as if the string isn't actually an IPv6 address we'll be able to handle the error gracefully.